src/twisted/web/script.py

```diff
@@ -161,7 +166,8 @@ def render(self, request):
         will NOT be handled with print - standard output goes to the log - but
         with request.write.
         """
-        request.setHeader(b"x-powered-by", networkString("Twisted/%s" % copyright.version))
+        request.setHeader(b"x-powered-by",
+            u"Twisted/{}".format(copyright.version).encode("utf-8"))
```

Comment:
[RFC 7230 says header values should be ASCII](https://tools.ietf.org/html/rfc7230#page-25).  Why has this been changed to UTF-8?

Reply:
copyright.version always has characters which fall under the ascii range when encoding with utf-8.